wiki/People/Users_with_unique_titles/en.md

```diff
@@ -272,13 +272,13 @@ These users were awarded their respective titles for having significantly contri
 
 | User | Title |
 | :-- | :-- |
-| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) | World Cup Organizer |
+| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) | World Cup Organiser |
```

Comment:
titles should be written as-is, and his (as well as other organi**s**ers') is `World Cup Organizer`